.github/workflows/tests_cpu_versions.yml

```diff
@@ -0,0 +1,33 @@
+on: [pull_request]
+
+jobs:
+  build-cpu-versions:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [3.6, 3.7, 3.8]
+        pytorch-version: [1.5.1, 1.6.0]
+        pytorch-channel: [pytorch]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup conda dependencies
+      uses: conda-incubator/setup-miniconda@v1
+      with:
+        auto-update-conda: true
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      shell: bash -l {0}
+      run: |
+        conda install curl make -c conda-forge
+        conda install pytorch=${{ matrix.pytorch-version }} cpuonly -c ${{ matrix.pytorch-channel }}
+        pip install -r requirements-dev.txt
+        make install
+    - name: Run Tests
+      shell: bash -l {0}
+      run: pytest -v --device cpu --dtype float32,float64 --cov=kornia --cov-report xml ./test --flake8 --mypy
```

Comment:
@edgarriba I don't know how many cores the CI platform provides, but in case more then one, we could run the tests parallel.

Reply:
According to the docs it has to so we could accelerate the tests a bit.

Reply:
I was checking `pytest-xdist` that have this features already
https://pypi.org/project/pytest-xdist/
`pytest -n NUMCPUS`

Reply:
it seems to cause a crash:
`
[gw0] linux Python 3.6.12 cwd: /home/runner/work/kornia/kornia
INTERNALERROR> Traceback (most recent call last):
INTERNALERROR>   File "/usr/share/miniconda/envs/test/lib/python3.6/site-packages/_pytest/main.py", line 189, in wrap_session
INTERNALERROR>     config.hook.pytest_sessionstart(session=session)
INTERNALERROR>   File "/usr/share/miniconda/envs/test/lib/python3.6/site-packages/pluggy/hooks.py", line 286, in __call__
INTERNALERROR>     return self._hookexec(self, self.get_hookimpls(), kwargs)
INTERNALERROR>   File "/usr/share/miniconda/envs/test/lib/python3.6/site-packages/pluggy/manager.py", line 93, in _hookexec
INTERNALERROR>     return self._inner_hookexec(hook, methods, kwargs)
INTERNALERROR>   File "/usr/share/miniconda/envs/test/lib/python3.6/site-packages/pluggy/manager.py", line 87, in <lambda>`

Reply:
Maybe with an earlier version? 1.3.x seems to work for me.

Reply:
weird - but seems to double the execution time :(